### PR TITLE
new board names

### PR DESCRIPTION
**Pull request by thomasjamatomics**

### CODE REVIEW

File: STM32/Libraries/Util/Inc/systemInfo.h [hunk 1]
```diff
@@ -7,12 +7,16 @@ typedef enum {
...
     Current           = 4,
-    FlowChip          = 5,
-    humidityChip      = 6,
+    GasFlow           = 5,
+    HumidityChip      = 6,
     Pressure          = 7,
     SaltFlowBoard     = 8,
     SaltLeak          = 9,
-    HotValvePumpBoard = 10
+    HotValveController = 10,
```
Comment (freddyrios) [inline STM32/Libraries/Util/Inc/systemInfo.h:15]: different indentention is surely spaces vs. tabs

**Review by freddyrios — APPROVED** [review]
hot valves using the new product type require LoopControl v3.100+, see https://github.com/copenhagenatomics/CA_LoopCode/pull/291

ps. LoopControl version is pending release due to github actions being down

**Comment by matias-cphatomics** [comment]
> hot valves using the new product type require LoopControl v3.100+, see [copenhagenatomics/CA_LoopCode#291](https://github.com/copenhagenatomics/CA_LoopCode/pull/291)
> 
> ps. LoopControl version is pending release due to github actions being down

Can the PR be merged now or are there still pending changes? :)

**Comment by freddyrios** [comment]
@matias-cphatomics yes, the updated LoopControl is already available

**Pull request merged**